INSTALLING.md

```diff
@@ -1,11 +1,12 @@
 ## Installing Plugins
 
-- [Installing OpenSearch Plugins](#installing-opensearch-plugins)
-- [Installing OpenSearch Dashboards Plugins](#installing-opensearch-dashboards-plugins)
+- [Installing Plugins](#installing-plugins)
+  - [Installing OpenSearch Plugins](#installing-opensearch-plugins)
+  - [Installing OpenSearch Dashboards Plugins](#installing-opensearch-dashboards-plugins)
 
 ### Installing OpenSearch Plugins
 
-Assemble, extract and run OpenSearch `1.0.0-beta1` using [Building OpenSearch package](https://github.com/opensearch-project/OpenSearch/blob/main/TESTING.asciidoc#creating-packages).  
+Assemble, extract and run OpenSearch `1.0.0-beta1` using [Building OpenSearch package](https://github.com/opensearch-project/OpenSearch/blob/main/TESTING.md#creating-packages).  
```

Comment:
Thanks for this. 

We should automatically find broken links, can you create an issue for this?
Quick googling: Github actions supports verifying broken links, https://github.com/marketplace/actions/broken-link-check. We could do something similar.

Reply:
created an issue: https://github.com/opensearch-project/opensearch-plugins/issues/29